arch/arm/boot/dts/overlays/revpi-flat-overlay.dts

```diff
@@ -117,30 +117,12 @@
 				brcm,function = <BCM2835_FSEL_GPIO_OUT>;
 				brcm,pull     = <BCM2835_PUD_OFF>;
 			};
-			tpm_pins {
-				/* irq reset */
```

Comment:
The order of the pins in the comment is wrong!
pin 0 is the reset
pin 1 is the irq

Reply:
Could you adjust the comment here, while you are on it?